Sambamba/0.6.8/Index.nf

```diff
@@ -2,7 +2,7 @@ process Index {
   tag {"SAMBAMBA_Index ${sample_id}"}
   label 'SAMBAMBA_0_6_8'
   label 'SAMBAMBA_0_6_8_Index'
-  clusterOptions = workflow.profile == "sge" ? "-l h_vmem=${params.index_mem}" : ""
+  clusterOptions = workflow.profile == "sge" ? "-l h_vmem=${params.sambambaindex.mem}" : ""
```

Comment:
change to params.mem

Reply:
fix

Reply:
params.mem :) not params.sambambaindex.mem

Reply:
ahhhhhh